TundraTD/Assets/Scripts/ModulesUI/MagicScreen/PlayerDeck.cs

```diff
@@ -10,20 +12,39 @@ namespace ModulesUI.MagicScreen
     /// </summary>
     public static class PlayerDeck
     {
+        public static BasicElement CurrentMostElement;
         public static ObservableCollection<BasicElement> DeckElements { get; } = new ObservableCollection<BasicElement>();
+        public static Dictionary<BasicElement, int> ElementsQuantity { get; } 
         public static Dictionary<BasicElement, Sprite> ElementIcons { get; }
 
         static PlayerDeck()
         {
-            var loadedIcons = Resources.LoadAll<Sprite>("Elements/");
+            var loadedIcons = new Sprite[6];
+            int i = 0;
+            foreach (var element in (BasicElement[])Enum.GetValues(typeof(BasicElement)))
+            {
+                loadedIcons[i] = Resources.Load<Sprite>($"Elements/{element}");
+                i++;
+            }
+            //var loadedIcons = Resources.LoadAll<Sprite>("Elements/");
             ElementIcons = new Dictionary<BasicElement, Sprite>
             {
-                [BasicElement.Air] = loadedIcons[1],
-                [BasicElement.Earth] = loadedIcons[2],
-                [BasicElement.Fire] = loadedIcons[3],
+                [BasicElement.None] = loadedIcons[0],
+                [BasicElement.Fire] = loadedIcons[1],
+                [BasicElement.Water] = loadedIcons[2],
+                [BasicElement.Earth] = loadedIcons[3],
                 [BasicElement.Lightning] = loadedIcons[4],
-                [BasicElement.Water] = loadedIcons[5],
-                [BasicElement.None] = null
+                [BasicElement.Air] = loadedIcons[5]
+            };
```

Comment:
If I remember, Resources.Load loaded files in alphabetic order. Now you can change this code to be sure that all elements have their codes correctly.

Reply:
You might wanted to say LoadAll<> but in my case I use Load<>(name), which loads the exact name value